src/game_constants.h

```diff
@@ -88,6 +88,18 @@
 /** Maximum range at which ranged attacks can be executed */
 #define RANGE_HARD_CAP 60
 
+/** Player view distance FLOAT */
+#define PLAYER_VIEW_DISTANCE 60.0f
+
+/** Player view distance INTEGER */
+#define PLAYER_VIEW_DISTANCE_INT 60
+
+/** Kind of related... but not to view distance exactly? Changing this would change balance of things I think? */
+#define DISTANCE_FROM_CENTER 60
+
+/** Was used based off lighting but I'm not sure?*/
+#define LIGHT_SIXTY_F 60.0f
```

Comment:
If you put the (current) value of the constant into its name, you might as well use the value directly. As soon as somebody changes the value, the name will be misleading.

Regarding two constants for the same value (int and float): define just one of them. And if you really need the other one (note: the compiler will convert the values as it needs it anyway), you should define it with a reference to the first one, e.g.
```C++
constexpr int DIST = 60;
constexpr float DIST_FLT = DIST;
```

But it's better to use the integer constant all the time, the compiler will automatically cast it *at compile time* when needed.

Reply:
Changed to using `constexpr`.  My goal was to replace hardcoded values with constants while being certain that it wouldn't have any side-effects.  I wanted to avoid side effects related to how integer division works vs floats, so I kept the types the same.

Reply:
Get rid of the int variant please.